sumo/plotting/phonon_bs_plotter.py

```diff
@@ -35,8 +36,18 @@ def __init__(self, bs, imag_tol=-5e-2):
         PhononBSPlotter.__init__(self, bs)
         self.imag_tol = imag_tol
 
+    def _plot_phonon_dos(self, dos_file, ax=None):
+        if ax is None:
+            ax = plt.gca()
+        y, x = np.genfromtxt('total_dos.csv', delimiter=',', unpack=True)
```

Comment:
The DOS data should be loaded in the `phonon_bandplot.py` script, with the data then passed through to the plotting object.